lib/nerves_hub_web/channels/device_socket.ex

```diff
@@ -15,7 +16,44 @@ defmodule NervesHubWeb.DeviceSocket do
   # Default 90 seconds max age for the signature
   @default_max_hmac_age 90
 
+  defoverridable init: 1, handle_info: 2, terminate: 2
+
+  @impl true
```

Comment:
```suggestion
  @impl Phoenix.Socket.Transport
```
Love me some explicit over implicit definitions 😉 https://hexdocs.pm/credo/Credo.Check.Readability.ImplTrue.html